types/framework/entities/actor.d.ts

```diff
@@ -210,17 +214,14 @@ declare class Actor<
    * Create a synthetic Actor using a provided Token instance
    * If the Token data is linked, return the true Actor entity
    * If the Token data is not linked, create a synthetic Actor using the Token's actorData override
-   * @param token -
-   * @returns
    */
   static fromToken(token: Token): Actor;
 
   /**
    * Create a synthetic Token Actor instance which is used in place of an actual Actor.
    * Cache the result in Actors.tokens.
-   * @param baseActor -
-   * @param token -
-   * @returns
+   * @param baseActor - The real actor to clone
+   * @param token - The Token containing the actor
```

Comment:
By style convention, these should be aligned on the dash.
```suggestion
   * @param baseActor - The real actor to clone
   * @param token     - The Token containing the actor
```